neard/src/lib.rs

```diff
@@ -49,7 +153,7 @@ fn init_logging(verbose: Option<&str>) {
         .init();
 }
 
-fn main() {
+pub fn neard_main() {
```

Comment:
I think we can bring back it to the `main.rs`. I expect the changes to `lib.rs` and `main.rs` to be mostly reverted.

Reply:
OK